src/types/fullvm.ts

```diff
@@ -0,0 +1,19 @@
+import VM, { Disk } from "./vm";
+import { v4 } from "uuid";
+import validateName from '../utils/validateName';
+import isValidInteger from '../utils/isValidInteger';
+
+export default class FullVM extends VM {
+  public name = `VM${v4().split("-")[0]}`;
+  public diskSize: number = 50;
+  public disks = [new Disk(undefined, undefined, this.diskSize, undefined)];
```

Comment:
I think the disk should be mounted on `/`

Reply:
done.